x/slashing/errors.go

```diff
@@ -37,3 +38,7 @@ func ErrValidatorNotJailed(codespace sdk.CodespaceType) sdk.Error {
 func ErrMissingSelfDelegation(codespace sdk.CodespaceType) sdk.Error {
 	return sdk.NewError(codespace, CodeMissingSelfDelegation, "validator has no self-delegation; cannot be unjailed")
 }
+
+func ErrSelfDelegationTooLow(codespace sdk.CodespaceType) sdk.Error {
```

Comment:
A validator can be jailed for dipping below their MinSelfBond.  To unjail themselves, they need to self delegate some more so that they are over the MinSelfBond again.  If they try to unjail without doing that, it will throw this error.

Reply:
yeah that makes sense, just the function name is not reflective of that.